integreat_cms/release_notes/current/unreleased/3819.yml

```
@@ -0,0 +1,2 @@
+en: Fix bug where saving a page could take forever and time out
+de: Fehler behoben, bei dem das Speichern einer Seite ewig dauern und zu einer Zeitüberschreitung führen konnte
```

```suggestion
de: Behebe den Fehler, bei dem das Speichern einer Seite zu einer Zeitüberschreitung führen konnte
```
Usually we write the release notes in imperative form